.github/workflows/add-to-project.yml

```diff
@@ -0,0 +1,18 @@
+name: Add new issues to project backlog
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          # You can target a repository in a different organization
+          # to the issue
+          project-url: https://github.com/orgs/ncihtan/projects/4
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
ADD_TO_PROJECT_PAT or PATH?

Reply:
`ADD_TO_PROJECT_PAT` is correct - it is secret token that the action needs to have permission to write in the project.

But this review reminds me that I actually need to create it in this repo (as this was copied from the data-models repo

Reply:
Ok, other than that, this looks good to go